src/tv2-common/actions/executeAction.ts

```diff
@@ -1973,4 +1989,19 @@ function executeActionClearGraphics<
 			tags: userData.sendCommands ? [TallyTags.GFX_CLEAR] : [TallyTags.GFX_ALTUD]
 		})
 	)
+
+	markPartAsModifiedByAction(context, 'current')
+}
+
+function markPartAsModifiedByAction(context: IActionExecutionContext, part: 'current' | 'next') {
+	const partInstance = context.getPartInstance(part)
+	if (!partInstance) {
+		return
+	}
+
+	if (!partInstance.part.metaData) {
+		context.updatePartInstance(part, {
+			metaData: literal<PartMetaData>({ ...(partInstance.part.metaData as PartMetaData), dirty: true })
+		})
+	}
```

Comment:
Could you explain this - if there's no `metaData`, you still spread it. And what if there is `metaData` - do we not set `dirty` then?

Reply:
Oh yeah, I've changed this to check for metadata not existing, then setting a default of `{}` 9f5e4dea055f89fe020d660e51aff21db00699ba